src/visit_vtk/full/vtkVisItTubeFilter.h

```diff
@@ -233,8 +247,8 @@ class VISIT_VTK_API vtkVisItTubeFilter : public vtkPolyDataAlgorithm
   double Theta;
 
 private:
-  vtkVisItTubeFilter(const vtkVisItTubeFilter&);  // Not implemented.
-  void operator=(const vtkVisItTubeFilter&);  // Not implemented.
+  vtkVisItTubeFilter(const vtkVisItTubeFilter&) = delete;
+  void operator=(const vtkVisItTubeFilter&) = delete;
```

Comment:
Same question as above...is `private:` scope still needed when using `= delete`. 

BTW...this seems like a new style of excluding such methods and maybe worth a quick announcement or something. I would have never known this is the new way of doing business.